lib/rake/build.rake

```diff
@@ -111,17 +111,13 @@ namespace :build do
         # The sequencing described here is the best for mitigating any issues
         # that may arise when that best practice is not followed.
         ChatClient.log 'Restarting <b>dashboard</b> Active Job worker(s).'
+        # Issue a stop command to all workers. Will kill if not stopped within ~20 seconds.
+        RakeUtils.system 'bin/delayed_job', 'stop'
+        # Start new workers in non-development environments
```

Comment:
super tiny nit: I got scared here because we don't want 10 workers on levelbuilder for example... I think the code itself is perfectly self-documenting and you can simply remove this comment. if you don't want to remove it entirely, "Start new workers" might be more appropriate.

Reply:
```suggestion
        # Start new workers
```

Gonna simplify the comment.